plugins/fluent-bit/functional-test/tests/tests.bats

```diff
@@ -0,0 +1,40 @@
+# shellcheck disable=SC2016
+
+: "${BATS_TEST_TIMEOUT:=120}"
+
+setup() {
+  bats_load_library bats-support
+  bats_load_library bats-assert
+  bats_load_library bats-tenzir
+}
+
+teardown() {
+  :
+}
```

Comment:
Is this the common header?

Reply:
For some reason we can't load the libraries in the `setup_suite` file, but we might be able to extract and `source` it. I'll look into it.